components/html_entities/nbsp.tsx

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// https://stackoverflow.com/a/54485712
+export default () => '\u00A0';
```

Comment:
And so does this one. Also, weird that we need to do this, but I guess it makes sense. I'm surprised we still use nbsps, but I assume a lot of them are from older code mostly.

Reply:
done